modules/apps/document-library/document-library-document-conversion/src/main/java/com/liferay/document/library/document/conversion/internal/DocumentConversionImpl.java

```diff
@@ -72,7 +79,7 @@ public class DocumentConversionImpl implements DocumentConversion {
 	public File convert(
 			String id, InputStream inputStream, String sourceExtension,
 			String targetExtension)
-		throws IOException {
+		throws Exception {
```

Comment:
This is a breaking change. Can you instead wrap the timeout exn in an io exn?

Reply:
👍🏻